test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -451,6 +452,13 @@ def test_transcript_scrubbing
     assert_scrubbed(@credit_card.verification_value.to_s, clean_transcript)
   end
 
+  def test_failed_xml_validation_unknown_card
+    assert response = @gateway.purchase(@amount, @sodexo_voucher, @options)
+    assert_failure response
+    assert_equal '5', response.error_code
+    assert_match %r{XML failed validation: Invalid payment details : Card number not recognised:}, response.message
+  end
```

Comment:
👍 Just to be extra safe, how about adding a test for `authorize` and `verify`?

Reply:
👍